src/less/day-slot.less

```diff
@@ -30,20 +31,25 @@
     min-height: 1em;
   }
 
+  .rbc-time-slot {
+    border-top: 1px solid lighten(@cell-border, 10%);
+  }
 }
 
 .rbc-time-slot {
   flex: 1 0 0;
-  min-height: 20px;
-  height: 20px;
 
-  & + & {
-    border-top: 1px solid @cell-border;
+  &.rbc-now {
+    font-weight: bold;
   }
+}
 
-  &:nth-child(even){
-    border-top-color: lighten(@cell-border, 10%);
-  }
+.rbc-timeslice-group {
+  border-bottom: 1px solid @cell-border;
+  min-height: 40px;
+  height: 40px;
```

Comment:
This is what I meant: https://github.com/intljusticemission/react-big-calendar/tree/full-height

![image](https://cloud.githubusercontent.com/assets/339286/15806973/772d0c00-2b1e-11e6-9bf0-db3a0e16ddc1.png)

![image](https://cloud.githubusercontent.com/assets/339286/15806977/8785707e-2b1e-11e6-809d-016019786620.png)